examples/arkadiko-data-indexing/vault-monitor/app/assets/config/manifest.js

```diff
@@ -1,4 +1,3 @@
 //= link_tree ../images
 //= link_directory ../stylesheets .css
-//= link_tree ../../javascript .js
-//= link_tree ../../../vendor/javascript .js
+//= link_tree ../../javascript .js
```

Comment:
```suggestion
//= link_tree ../../javascript .js

```
(the diff editor was complaining that it's missing a newline at the end of the file)